tests/framework/utils_cpu_templates.py

```diff
@@ -24,23 +24,23 @@ def get_supported_cpu_templates():
     Return the list of CPU templates supported by the platform.
     """
     # pylint:disable=too-many-return-statements
-    match cpuid_utils.get_cpu_vendor():
-        case cpuid_utils.CpuVendor.INTEL:
-            # T2CL template is only supported on Cascade Lake and newer CPUs.
-            if global_props.cpu_codename == cpuid_utils.CpuModel.INTEL_SKYLAKE:
-                return sorted(set(INTEL_TEMPLATES) - set(["T2CL"]))
+    vendor = get_cpu_vendor()
+    cpu_model = global_props.cpu_model
+    host_linux = global_props.host_linux_version_tpl
+    cpu_codename = global_props.cpu_codename
```

Comment:
Since those are repeated, they could move to globals or just leave as-is, they are only used once

Reply:
Its a minor thing but I prefer:
```python
    vendor = get_cpu_vendor()
    cpu_model = global_props.cpu_model
    host_linux = global_props.host_linux_version_tpl
    cpu_codename = global_props.cpu_codename

    match (vendor, cpu_codename, cpu_model):
        # T2CL template is only supported on Cascade Lake and newer CPUs.
        case (CpuVendor.INTEL, CpuModel.INTEL_SKYLAKE, _):
```
over
```python
    match (
        get_cpu_vendor(),
        global_props.cpu_codename,
        global_props.cpu_model
    ):
        # T2CL template is only supported on Cascade Lake and newer CPUs.
        case (CpuVendor.INTEL, CpuModel.INTEL_SKYLAKE, _):
```
I'm not strongly opinionated either way.

Reply:
it should be (here and below):

```
    match global_props.cpu_vendor, global_props.cpu_codename:
```

which fits in one line